### ISSUE TITLE
model.py中的39行 注意力机制的代码有错误

### ISSUE THREAD

**Issue by lmx760581375** [adtalos/AITM-torch#1]
a = torch.sum(torch.mul(Q, V), -1) / torch.sqrt(torch.tensor(self.dim)) 中的V应该改为K

**Comment by rockyzhengwu** [comment]
@lmx760581375 多谢指正
